### ISSUE TITLE
KeyError multiValueQueryStringParameters

### ISSUE THREAD

**Issue by dipthegeezer** [aws/chalice#1154]
Hi

Upgrading my project to use the latest version of chalice I get the following error:

`{
  "errorType": "KeyError",
  "errorMessage": "'multiValueQueryStringParameters'",
  "stackTrace": [
    "  File \"/var/task/chalice/app.py\", line 829, in __call__\n    event['multiValueQueryStringParameters'],\n"
  ]
}`

I'm using the latest version of aws sam cli and running my code locally. My code worked previously  with the 1.8.0 release:

```
@app.route('/')
def index() -> Dict:
    message = get_message()
    return message


def get_message() -> Dict:
    return {"message": "hit me up."}
```

Digging deeper as far as I can seen multiValueQueryStringParameters is not a valid key in the event object when run with aws sam local. Possibly I should file a bug under that project but I would like confirmation before I do so since I can see that this work was recently merged in here.

Thank you.

**Comment by kapilt** [comment]
seems related to #1132

**Comment by stealthycoin** [comment]
This looks like a bug in SAM to me, would be good to raise with them. It was indeed introduced by #1132. However according to API Gateway that value is always populated so we shouldn't have to worry about this.

**Comment by stealthycoin** [comment]
The most recent version of SAM has fixed this issue. Should be fixed 0.18.0.
